src/components/css/bookStyle.css

```diff
@@ -1,8 +1,13 @@
+* {
+  font-family: 'Montserrat', sans-serif;
+}
+
 .container {
-  background-color: #9cecec;
+  background-color: #eee;
   display: flex;
   flex-direction: column;
   gap: 12px;
+  padding: 10px 0;
 }
```

Comment:
- You have done well styling your project so far, but there are some differences between your style and the given design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please reference the picture I attached below.

Your project:
1. Please increase the spacing between the elements for the book details, just as it is in the design template. 👍 👍 
2. There is a missing vertical line.
3. Please increase the spacing of the book progress information 😄 
![gnfgnxfgn](https://github.com/AliShahSafdari/bookstore/assets/73607512/7feadb25-96fd-463f-ba28-97eb8a94aa45)

Design template:
![sehserhs](https://github.com/AliShahSafdari/bookstore/assets/73607512/cd5cf504-32ed-450f-9ae4-7266213a5ceb)

Reply:
Thanks